docs/CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
  - Fixed a bug which made it impossible to re-send the same transaction after abandoning it while manually changing the nonce.
 
+### Changed
+ - Bumping batch size defaults for EVM specific configuration. If you are overriding any of these fields in your local config, please consider if it is necesssary:
+  - LogBackfillBatchSize: 1000
+  - RPCDefaultBatchSize: 250
+  - [EVM.GasEstimator.BlockHistory].BatchSize: 100
```

Comment:
Let's use the toml syntax for these. And does that bracket format actually work?
```suggestion
  - `LogBackfillBatchSize = 1000`
  - `RPCDefaultBatchSize = 250`
  - `EVM.GasEstimator.BlockHistory.BatchSize = 100`
```
It's a little weird b/c EVM is a list :thinking:

Reply:
Leaves the door open for running multiple chains on one node I suppose 🤷

Reply:
Yeah that's the plan - I'm just not sure how to document the field name of a toml array table: https://toml.io/en/v1.0.0#array-of-tables
If this dot syntax is actually valid, then it should also have to contain the index of the EVM array table:
`EVM.<index>.GasEstimator.BlockHistory.BatchSize`
I think as-is is clear though.

Reply:
Yep, was looking at the same reference earlier - the dot syntax is valid. 

On the last cycle I left `EVM` out from `GasEstimator.Block...` so that I didn't have the messiness of the 0 index 🤠